### ISSUE TITLE
toJSON or JSON.stringify is unresponsive

### ISSUE THREAD

**Issue by adefokun** [parse-community/Parse-SDK-JS#415]
When I call toJSON or JSON.stringify in Javascript SDK the application becomes unresponsive.

**Comment by andrewimm** [comment]
Can you provide code that reproduces this? Also, information on the environment would be helpful (node.js vs browser, which version)

**Comment by adefokun** [comment]
Thank you. 
    
```
        Parse.Cloud.run('getAttendees', {},{
            success: function(d){ 
                var data = JSON.stringify(d);
            },
            error: function(error){
            }
        });
```
 "parse-server": "^2.3.6",
"express": "^4.13.4",
"node": "4.4",

**Comment by adefokun** [comment]
No errors are indicated but the application becomes unresponsive, every time.

**Comment by adefokun** [comment]
I think it goes into an infinite loop.

**Comment by adefokun** [comment]
I notice there are recursive loops because the server now automatically include all pointers. How can this functionality be disabled so that I can manually include pointers that I want included in a query?

**Comment by adefokun** [comment]
I closed the issue thinking the problem was with parse-server. I noticed that the details of pointers in arrays are included by default, I did not have to use "include" and the server actually returned pointers, but the SDK populated them with details.

```
var interestAreasQuery = new Parse.Query("InterestArea");
interestAreasQuery.find().then(function(result){
   console.log(result);
}}
```
IntrestedArea has an interest key, which is an array of pointers. The details of each pointer is included. And because interest has a pointer interestedArea, the pointer details is also included and the leads to infinite loop. In a situation like this JSON.stringify(result) renders the application unresponsive.
